pkg/app/app_fetch.go

```diff
@@ -8,11 +8,17 @@ import (
 	"fmt"
 	"path"
 	"strconv"
+	"strings"
 	"time"
 
 	"github.com/vmware-tanzu/carvel-kapp-controller/pkg/exec"
 )
 
+const (
+	addCACertMissingHintMsg = "(hint: The CA Certificate from URL is unknown/invalid. Add valid CA certificate to the kapp-controller configuration to reconcile successfully)"
+	caCertMissingError      = "x509: certificate signed by unknown authority"
```

Comment:
The day we move to an interface from the binaries to KC will be soooo good. Remove the need for relying on output like this. Not a comment on this PR, so much as the state of the world with our tools. LGTM!